docs/docs/shards/index.md

```diff
@@ -559,4 +559,101 @@ A `Float4` type value looks like this: `(Float4 -8.84 38.2 4.7 0.4)`.
     [info] [2022-07-22 22:23:24.076] [T-25152] [logging.cpp::55] [mywire] (9.9, 9.9, 10.1, 9.9)
     ```
 
+### Bytes
```

Comment:
This whole thing is wrong.
`Bytes` type is just an array of bytes.

Reply:
The output of the `(Log)` had me confused.
Fixed it in commit [#c625d64213887817a485f8e81a5b2ac69646e765](https://github.com/fragcolor-xyz/shards/pull/325/commits/c625d64213887817a485f8e81a5b2ac69646e765).